br/use-br-command-line-tool.md

```diff
@@ -244,6 +244,24 @@ LAST_BACKUP_TS=`br validate decode --field="end-version" -s local:///home/tidb/b
 
 In the above example, for the incremental backup data, BR records the data changes and the DDL operations during `(LAST_BACKUP_TS, current PD timestamp]`. When restoring data, BR first restores DDL operations and then the data.
 
+### Point in Time recovery (experimental feature)
+
+Where full backups and incremental backups allow you to the time of the backup was taken, Point in Time recovery allows you to restore to a point in time of your choosing.
+
+An example scenario would be to take a full backup every day and take incremental backups every 6 hours and then use TiCDC for Point in Time recovery. For this example assume the full backup was taken at 00:00 and the first incremental was taken at 06:00. If you now would like to restore the database to the state of 07:16 you would first restore the full backup and the incremental backup and finally you would use TiCDC logs to fill in the gap between 06:00 and 07:16.
+
+Restoring the TiCDC logs can be done like this:
+
+1. Restore a full backup with `br restore full`
+2. Restore incremental backup(s) (optional)
+3. Use `br restore cdclog` to restore the transactions that happened after the last incremental backup.
+
+```shell
+br restore cdclog --storage local:///data/cdclog --start-ts $START_TS --end-ts $END_TS
+```
+
+Here `local:///data/cdclog` is the location of the TiCDC logs. This could be on the local filesystem or on external storage like S3. The `$START_TS` is the end position of the restore from the last restored backup (either a full backup or an incremental backup). The `$END_TS` is the point to which you want to restore your data.
```

Comment:
```suggestion
    In the command above:

        - `local:///data/cdclog` is the location of the TiCDC logs. This might be on the local filesystem or on the external storage like S3.
        - `$START_TS` is the end position of the restore from the last restored backup (either a full backup or an incremental backup).
        - `$END_TS` is the point to which you want to restore your data.
```